docs/html-css/week-1/lesson.md

```diff
@@ -415,30 +415,35 @@ The following table contains some of the most commonly used CSS units.
   <thead>
     <tr>
       <th>Unit</th>
+      <th>Name</th>
       <th>Relative or absolute</th>
       <th>Description</th>
     </tr>
   </thead>
   <tbody>
+    <tr>
+      <td><code>px</code></td>
+      <td>pixel</td>
+      <td>Absolute</td>
+      <td>This used to refer to real physical pixels on screens, but on modern devices with high resolution displays one CSS pixel is usually rendered using a whole multiple of real pixels. For rendering on screens, a CSS “reference pixel” is defined as the visual angle of one pixel on a device with a pixel density of 96 pixels per inch at a distance of an arm’s length from the reader.</td>
```

Comment:
So this isn't wrong, but I don't think this is useful to lead with on day 1 module 1. Is it possible you may be falling prey to the [expert blind spot](https://teachtogether.tech/en/index.html#s:memory-becoming-expert)? (the tendency of experts to organize explanation according to the subject’s deep principles rather than being guided by what their learners already know)

Let's instead think about what the trainee needs to know to use a pixel. Do they need to use a pixel? Is a pixel the right thing to lead with? Where would this declaration about pixel rendering fit in a c[oncept map](https://teachtogether.tech/en/index.html#s:conceptmaps) of this lesson?

I really like this video on motivating explanations for trainees https://teachertraining.codeyourfuture.io/content/how-do-i-motivate-explanations-for-students